modules/openxr/doc_classes/OpenXRInterface.xml

```diff
@@ -108,8 +108,8 @@
 		<method name="is_foveation_supported" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if OpenXR's foveation extension is supported, the interface must be initialized before this returns a valid value.
-				[b]Note:[/b] This feature is only available on the Compatibility renderer and currently only available on some stand alone headsets. For Vulkan set [member Viewport.vrs_mode] to [code]VRS_XR[/code] on desktop.
+				Returns [code]true[/code] if OpenXR's foveation extension is supported. The interface must be initialized before this returns a valid value.
+				[b]Note:[/b] This feature is only available on the Compatibility renderer and currently only available on some standalone headsets. For Vulkan set [member Viewport.vrs_mode] to [code]VRS_XR[/code] on desktop.
```

Comment:
Oo, this line isn't true anymore. I guess I should probably do this in another PR, but the truth is more like this:

```suggestion
				[b]Note:[/b] On Vulkan, also set [member Viewport.vrs_mode] to [code]VRS_XR[/code].
```